src/main/resources/templates/listeArtists.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/header :: head">
+
+</head>
+<body>
+  <div th:insert="fragments/header :: menu"> </div>
+
+
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <h1>Liste des artistes</h1>
+          <a th:href="|artist/new|" class="btn btn-primary">Nouvel artiste</a>
+          <table class="table table-hover table-striped">
+            <thead>
+              <tr>
+                <th scope="col">Nom
+                  <span class="glyphicon glyphicon-chevron-up"></span>
+                </th>
+                <th scope="col"></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr th:each="artists : ${artist}">
+                <th scope="row" th:text="${artists.name}">A Cor Do Som</th>
+                <td><a th:href="|/artists/${artists.id}|" class="btn btn-primary">Détail</a></td>
+              </tr>
+            </tbody>
+          </table>
+          <div class="row">
+            <div class="col-lg-6">
+              <p>Affichage des artistes [[${start}]] à [[${end}]] sur un total de <span th:text="${artist.getTotalElements()}"></span></p>
+            </div>
+            <div class="col-lg-6">
+              <ul class="pagination">
+                <li class="disabled" th:if="${page &gt; 1}">
+                  <a th:href="|/artists?page=${page - 2}&size=10&sortProperty=name&sortDirection=ASC|">«</a>
+                </li>
+                <li><a href="#">Page 1</a></li>
```

Comment:
Ici remplace le 1 en dur par la valeur dynamique
```suggestion
                <li><a href="#">Page [[${page}]]</a></li>
```